packages/roosterjs-editor-plugins/lib/plugins/TableResize/TableResize.ts

```diff
@@ -377,13 +393,46 @@ export default class TableResize implements EditorPlugin {
         }
     }
 
-    private createResizer(
+    private setTableResizer(rect: null): void;
+    private setTableResizer(rect: Rect): void;
+    private setTableResizer(rect: Rect | null): void {
+        // remove old one if exists
+        while (this.tableResizerContainer.hasChildNodes()) {
```

Comment:
null check for tableResizerContainer?

Reply:
tableResizerContainer is created in initialize() and removed in dispose(), so it should exist when we call this method.